.github/dependabot.yml

```
@@ -0,0 +1,40 @@
+      patch-updates:
+        patterns:
+          - "*"
+        update-types:
+          - "patch"
+    # Customize commit messages
+    commit-message:
+      prefix: "Updated:"
+      prefix-development: "Updated:"
+      include: "scope"
+    # Allow automatic rebasing when conflicts occur
+    rebase-strategy: "auto" 
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)